src/site/layouts/tests/vamc/health_care_local_facility.unit.spec.js

```diff
@@ -24,5 +24,32 @@ describe('health_care_local_facility', () => {
     it('should not render .mental-health-clinic-phone', () => {
       expect(container.querySelector('.mental-health-clinic-phone')).to.be.null;
     });
+
+    // Default state, main phone number is in dataset
+    it('should render main phone number', () => {
+      expect(container.querySelector('.main-phone a').textContent).to.equal(
+        data.fieldPhoneNumber,
+      );
+    });
+
+    // Test return if the fieldPhoneNumber string is empty
+    it('should not render .main-phone if string is empty', async () => {
+      // Clear fieldPhoneNumber to mimic an entry error
+      data.fieldPhoneNumber = '';
```

Comment:
It would be better to create a copy of the test data so tests don't affect each other. e.g.

```js
const testData = { ...data, fieldPhoneNumber: ''};
```

(I haven't tested that but hopefully you get the idea!)

Reply:
ah yeah good point